testonedrivesdk/test_streams.py

```diff
@@ -28,7 +28,7 @@ def test_put(self, MockHttpProvider, MockAuthProvider):
         auth_provider = onedrivesdk.AuthProvider()
         client = onedrivesdk.OneDriveClient("onedriveurl/", http_provider, auth_provider)
 
-        response_item = client.drives["me"].items["root"].children["newFile.txt"].content.request().upload("./myPath/myFile.txt")
```

Comment:
in order to make this backward compatible need to reverse the content and path arguments